src/main/java/uk/gov/companieshouse/web/accounts/service/transaction/impl/TransactionServiceImpl.java

```diff
@@ -143,4 +144,9 @@ public boolean isPayableTransaction(String transactionId, String companyAccounts
             throw new ServiceException("Invalid URI for fetching transactions resource", e);
         }
     }
+
+    @Override
+    public Transaction getTransaction(String uri) throws ApiErrorResponseException, URIValidationException {
```

Comment:
Can we have this method throw only `ServiceException`'s so the places which call this method don't need to know about `ApiErrorResponseException` or `URIValidationException`?

Reply:
Changed the code to throw ServiceException